server/src/controllers/Events.js

```diff
@@ -46,11 +46,23 @@ module.exports = {
 		}
 	},
 	async updateEvent(req, res) {
-		const eventId = req.params.id;
-		const event = await Event.findById(eventId);
-		res.json({
-			"id": event._id
-		});
+		try {
+			const file = req.file;
+			const image = cloudinary.v2.uploader.upload(file.path);
```

Comment:
Is uploading a file compulsory ? It'd be nice if you can put a handler here. Just to handle the case if there's no file. If at all we decide to not compulsorily upload files, I'm afraid this bit of code will face some problems.

Reply:
Yes, I added the check if the file is not uploaded in updateEvent and in addEvent function call. So now uploading the image is not compulsory,